scripts/climate_baseline/load-climate-baseline-values.py

```diff
@@ -0,0 +1,39 @@
+import sys
```

Comment:
Is there a need for this file to be in its own subdir, or would the scripts/ directory be sufficient?